optimum/intel/neural_compressor/trainer.py

```diff
@@ -141,10 +142,17 @@ def __init__(
         self.deepspeed = None
 
         # Attach dtype and architecture to the config
-        self.dtype = "int8" if quantization_config is not None else str(get_parameter_dtype(self.model)).split(".")[1]
+        if quantization_config is not None:
+            self.dtype = "int8"
+            self.model.config.backend = quantization_config.backend
+        else:
+            if isinstance(self.model, INCBaseModel):

```

Comment:
In which case would the model be a `INCBaseModel` ?

Reply:
Is there a possibility that when we load a quantified jit model with INCModelForCausalLM and use a trainer for evaluation?

Reply:
I don't see why this integration would be adapted as the trainer cannot perform the training step using a quantized jit model.